docs/source/1.0/guides/generating-cloudformation-resources.rst

```diff
@@ -0,0 +1,484 @@
+            "cloudformation": {
+                "service": "smithy.example#Queues",
+                "organizationName": "Smithy"
+            }
+        }
+    }
+
+.. important::
+
+    A buildscript dependency on "software.amazon.smithy:smithy-aws-cloudformation:__smithy_version__" is
+    required in order for smithy-build to map the "cloudformation" plugin name
+    to the correct Java library implementation.
+
+
+-------------------------------------
+CloudFormation configuration settings
+-------------------------------------
+
+The ``cloudformation`` plugin provides configuration options to influence the
+Resource Schemas that it generates.
+
+.. tip::
+
+    You typically only need to configure the ``service`` and
+    ``organizationName`` settings to generate Resource Schemas.
+
+The following settings are supported:
+
+.. _generate-cloudformation-setting-service:
+
+service (``string``)
+    **Required**. The Smithy service :ref:`shape ID <shape-id>` to convert.
+    For example, ``smithy.example#Queues``.
+
+    .. code-block:: json
+
+        {
+            "version": "1.0",
+            "plugins": {
+                "cloudformation": {
+                    "service": "smithy.example#Queues",
+                    "organizationName": "Smithy"
+                }
+            }
+        }
+
+.. _generate-cloudformation-setting-organizationName:
+
+organizationName (``string``)
+    The ``Organization`` component of the resource's `type name`_. Defaults to
+    "AWS" if the :ref:`aws.api#service-trait` is present, otherwise is
+    **required**.
+
+    .. code-block:: json
+
+        {
+            "version": "1.0",
+            "plugins": {
+                "cloudformation": {
+                    "service": "smithy.example#Queues",
+                    "organizationName": "Smithy"
+                }
+            }
+        }
+
+.. _generate-cloudformation-setting-serviceName:
+
+serviceName (``string``)
+    Allows overriding the ``Service`` component of the resource's `type name`_.
+    This value defaults to the :ref:`cloudFormationName property of the
+    aws.api#service trait <service-cloudformation-name>` if present, or the
+    shape name of the specified service shape otherwise.
+
+    .. code-block:: json
+
+        {
+            "version": "1.0",
+            "plugins": {
+                "cloudformation": {
+                    "service": "smithy.example#QueueService",
+                    "organizationName": "Smithy",
+                    "serviceName": "Queues"
+                }
+            }
+        }
+
+.. _generate-cloudformation-setting-externalDocs:
+
+externalDocs (``[string]``)
+    Limits the source of generated `"documentationUrl" fields`__ to the
+    specified priority ordered list of names in an :ref:`externaldocumentation-trait`.
+    This list is case insensitive. By default, this is a list of the following
+    values: "Documentation Url", "DocumentationUrl", "API Reference", "User
```

Comment:
Are these case-sensitive? Because "Url" vs "URL" in a free-form field like this is asking for trouble I think.

Reply:
Sentence before it says "This list is case insensitive."